pkg/internal/controller/controller.go

```diff
@@ -169,6 +169,7 @@ func (c *Controller) Start(stop <-chan struct{}) error {
 
 		// Launch workers to process resources
 		c.Log.Info("Starting workers", "worker count", c.MaxConcurrentReconciles)
+		ctrlmetrics.WorkerCount.WithLabelValues(c.Name).Set(float64(c.MaxConcurrentReconciles))
```

Comment:
Can you explain the logic behind collecting this metric?

Reply:
* It allows correlating other metrics to the number of workers, for example workqueue_depth
* It allows calculating the `available_workers`